lib/spack/spack/build_systems/cmake.py

```diff
@@ -355,6 +357,13 @@ def std_cmake_args(self):
         args += getattr(self.pkg, "cmake_flag_args", [])
         return args
 
+    @staticmethod
+    def setup_dependent_cmake_package(module, dependent_spec, *propagated_args):
```

Comment:
This doesn't use `module`, so it shouldn't be an arg

Reply:
Makes sense!